### PR TITLE
Adds FAQ to community repo

### PR DESCRIPTION
**Pull request by stevsmit**

### CODE REVIEW

**Review by lhawthorn — COMMENTED** [review]
Quick review only, needs another pair of eyes. Would appreciate one of the maintainers reviewing.

**Review by joesepi — APPROVED** [review]
I haven't scrutinized the information presented here thoroughly but in general, I approve this FAQ. I do wonder why we are doing a doc for the community FAQ but are doing a discussion topic for the other repos.

**Pull request merged**
